services/QuillLMS/app/services/demo/create_admin_report.rb

```diff
@@ -70,7 +73,7 @@ def reset
       teacher = find_or_create_teacher_data(row['Teacher'], school, subscription)
 
       # create classroom data
-      classroom = Classroom.create(name: row['Classroom'])
+      classroom = Classroom.create(name: row['Classroom'], grade: (GRADE_MIN..GRADE_MAX).to_a.sample.to_s)
       all_classrooms.push(classroom)
       ClassroomsTeacher.create(classroom: classroom, user: teacher, role: ClassroomsTeacher::ROLE_TYPES[:owner])
       student_names = (1..NUMBER_OF_STUDENTS_PER_CLASSROOM).to_a.map { |i| "#{Faker::Name.first_name} #{Faker::Name.last_name}" }
```

Comment:
I realize this is older code, but `times` here is more idiomatic ruby:

`student_names = NUMBER_OF_STUDENTS_PER_CLASSROOM.times.map  { "#{Faker::Name.first_name} #{Faker::Name.last_name}" }`